api/v1beta1/groupversion_info.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package v1beta1 contains API Schema definitions for the authorino-operator v1beta1 API group
+//+kubebuilder:object:generate=true
+//+groupName=operator.authorino.kuadrant.net
+package v1beta1
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/scheme"
+)
+
+var (
+	// GroupVersion is group version used to register these objects
+	GroupVersion = schema.GroupVersion{Group: "operator.authorino.kuadrant.net", Version: "v1beta1"}
```

Comment:
Sorry, @jjaferson. It should be [`*.kuadrant.io`](https://github.com/Kuadrant/kuadrant-controller/blob/1d691f6a792cfc4bc0f39de7de3e6c2beba5488c/apis/networking/v1beta1/groupversion_info.go#L29).

```suggestion
	GroupVersion = schema.GroupVersion{Group: "operator.authorino.kuadrant.io", Version: "v1beta1"}
```